Eatery/Controllers/Eateries/Campus/CampusEateryExpandedMenuViewControllerswift.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CampusEateryExtendedMenuViewController.swift
+//  Eatery
+//
+//  Created by Sergio Diaz on 12/20/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class CampusEateryExpandedMenuViewController: UIViewController {
```

Comment:
Should this be converted into a view? It's usage feels much more like a view to be honest